<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Other_Editer</class>
 <widget class="QDialog" name="Other_Editer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="BaseTab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string/>
           </property>
           <widget class="QLabel" name="label">
            <property name="geometry">
             <rect>
              <x>21</x>
              <y>20</y>
              <width>24</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>通道</string>
            </property>
           </widget>
           <widget class="QComboBox" name="ChannelCBX">
            <property name="geometry">
             <rect>
              <x>102</x>
              <y>21</y>
              <width>69</width>
              <height>20</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="label_2">
            <property name="geometry">
             <rect>
              <x>21</x>
              <y>60</y>
              <width>41</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>平台ID</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="SubPrjNameID">
            <property name="geometry">
             <rect>
              <x>101</x>
              <y>60</y>
              <width>133</width>
              <height>20</height>
             </rect>
            </property>
           </widget>
           <widget class="QLineEdit" name="SubPrjName">
            <property name="geometry">
             <rect>
              <x>101</x>
              <y>90</y>
              <width>133</width>
              <height>20</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="label_17">
            <property name="geometry">
             <rect>
              <x>21</x>
              <y>90</y>
              <width>41</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>点位名</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="SubPrjNum">
            <property name="geometry">
             <rect>
              <x>100</x>
              <y>120</y>
              <width>133</width>
              <height>20</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="label_18">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>120</y>
              <width>51</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>分项编号</string>
            </property>
           </widget>
           <widget class="QLabel" name="Linklable">
            <property name="geometry">
             <rect>
              <x>260</x>
              <y>90</y>
              <width>91</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>获取平台参数</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>网络设置</string>
           </property>
           <widget class="QLabel" name="label_11">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>30</y>
              <width>71</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>IP Address</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="IP_Address">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>30</y>
              <width>371</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>127.0.0.1</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_12">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>70</y>
              <width>71</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>Server Port</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="ServerPortNum">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>90</y>
              <width>51</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>502</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="RetryCount">
            <property name="geometry">
             <rect>
              <x>140</x>
              <y>90</y>
              <width>51</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>2</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_13">
            <property name="geometry">
             <rect>
              <x>140</x>
              <y>70</y>
              <width>71</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>重发次数</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="RetryInternal">
            <property name="geometry">
             <rect>
              <x>250</x>
              <y>90</y>
              <width>51</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>2</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_14">
            <property name="geometry">
             <rect>
              <x>250</x>
              <y>70</y>
              <width>71</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>重发间隔(s)</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="HeartInternal">
            <property name="geometry">
             <rect>
              <x>360</x>
              <y>90</y>
              <width>51</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>1</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_15">
            <property name="geometry">
             <rect>
              <x>360</x>
              <y>70</y>
              <width>71</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>心跳间隔(s)</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>设备信息</string>
           </property>
           <widget class="QLabel" name="label_4">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>100</y>
              <width>51</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>站点编号</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="StationID">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>100</y>
              <width>90</width>
              <height>20</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="label_9">
            <property name="geometry">
             <rect>
              <x>20</x>
              <y>40</y>
              <width>51</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>设备编号</string>
            </property>
           </widget>
           <widget class="QWidget" name="layoutWidget">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>36</y>
              <width>351</width>
              <height>22</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLineEdit" name="DeviceID7"/>
             </item>
             <item>
              <widget class="QLineEdit" name="DeviceID6"/>
             </item>
             <item>
              <widget class="QLineEdit" name="DeviceID5"/>
             </item>
             <item>
              <widget class="QLineEdit" name="DeviceID4"/>
             </item>
             <item>
              <widget class="QLineEdit" name="DeviceID3"/>
             </item>
             <item>
              <widget class="QLineEdit" name="DeviceID2"/>
             </item>
             <item>
              <widget class="QLineEdit" name="DeviceID1"/>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="SysTab">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>系统</string>
           </property>
           <widget class="QComboBox" name="CommProtocol">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>30</y>
              <width>69</width>
              <height>20</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="label_7">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>30</y>
              <width>61</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>通信协议</string>
            </property>
           </widget>
           <widget class="QComboBox" name="CommWay">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>60</y>
              <width>69</width>
              <height>20</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="label_8">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>60</y>
              <width>61</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>通信方式</string>
            </property>
           </widget>
           <widget class="QWidget" name="layoutWidget_4">
            <property name="geometry">
             <rect>
              <x>100</x>
              <y>180</y>
              <width>351</width>
              <height>22</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLineEdit" name="RTC_Timer7"/>
             </item>
             <item>
              <widget class="QLineEdit" name="RTC_Timer6"/>
             </item>
             <item>
              <widget class="QLineEdit" name="RTC_Timer5"/>
             </item>
             <item>
              <widget class="QLineEdit" name="RTC_Timer4"/>
             </item>
             <item>
              <widget class="QLineEdit" name="RTC_Timer3"/>
             </item>
             <item>
              <widget class="QLineEdit" name="RTC_Timer2"/>
             </item>
             <item>
              <widget class="QLineEdit" name="RTC_Timer1"/>
             </item>
            </layout>
           </widget>
           <widget class="QLabel" name="label_25">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>184</y>
              <width>51</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>RTC时钟</string>
            </property>
           </widget>
           <widget class="QLabel" name="label_26">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>90</y>
              <width>81</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>GPRS传输模式</string>
            </property>
           </widget>
           <widget class="QComboBox" name="GPRS_Pass_Mode">
            <property name="geometry">
             <rect>
              <x>110</x>
              <y>90</y>
              <width>69</width>
              <height>20</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="label_27">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>150</y>
              <width>81</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>密度</string>
            </property>
           </widget>
           <widget class="QLineEdit" name="Density">
            <property name="geometry">
             <rect>
              <x>108</x>
              <y>150</y>
              <width>90</width>
              <height>20</height>
             </rect>
            </property>
            <property name="text">
             <string>426.00</string>
            </property>
           </widget>
           <widget class="QComboBox" name="SystemRestart">
            <property name="geometry">
             <rect>
              <x>108</x>
              <y>120</y>
              <width>71</width>
              <height>20</height>
             </rect>
            </property>
           </widget>
           <widget class="QLabel" name="label_28">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>120</y>
              <width>81</width>
              <height>16</height>
             </rect>
            </property>
            <property name="text">
             <string>重启</string>
            </property>
           </widget>
           <widget class="QPushButton" name="getPC_TimeBtn">
            <property name="geometry">
             <rect>
              <x>460</x>
              <y>180</y>
              <width>81</width>
              <height>23</height>
             </rect>
            </property>
            <property name="text">
             <string>获取PC时间</string>
            </property>
           </widget>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="ConfirmBtn">
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelBtn">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
